modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx

```diff
@@ -217,22 +216,74 @@ const OrderableTableRow = ({
 	);
 };
 
+const Table = ({
+	actions,
+	fields,
+	items,
+	onDragCrossover,
+	onDrop,
+	query,
+}: {
+	actions?: Array<IAction>;
+	fields: Array<IField>;
+	items: Array<any>;
+	onDragCrossover: Function;
+	onDrop: Function;
+	query: string;
+}) => {
```

Comment:
nit: should this be moved to an interface?

Maybe we don't need a rule for this, but not sure if it makes more sense to keep the Prop type declarations in their own interface instead of inline

Reply:
Lately I'm leaning more towards keeping interfaces inline, if they are used only once. Having then separate definitely makes sense if they are used multiple times, or exported.